internal/service/alarms/internal/serviceconfig/serviceconfig.go

```diff
@@ -0,0 +1,300 @@
+package serviceconfig
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"strconv"
+	"time"
+
+	api "github.com/openshift-kni/oran-o2ims/internal/service/alarms/api/generated"
+	serverutils "github.com/openshift-kni/oran-o2ims/internal/service/common/utils"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/openshift-kni/oran-o2ims/internal/controllers/utils"
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/db/models"
+	"github.com/stephenafamo/bob/dialect/psql"
+	"github.com/stephenafamo/bob/dialect/psql/dm"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	cleanupCronJobName   = "alarms-server-events-cleanup"
+	cleanupConfigMapName = "alarms-server-events-cleanup-sql"
+	cleanScriptDir       = "/scripts"
+	cleanScriptName      = "cleanup.sql"
+)
+
+var (
+	cronSchedule    = "0 * * * *" // run at the start of every hour e.g 2:00, 3:00 etc
+	jobBackoffLimit = int32(3)
+	successLimit    = int32(3)
+	failureLimit    = int32(1)
+	commonLabels    = map[string]string{
+		"app": "alarms-server",
```

Comment:
We should use our server name constants from utils.

Reply:
using constant